packages/ra-ui-materialui/src/layout/Title.tsx

```diff
@@ -4,10 +4,17 @@ import { ReactElement } from 'react';
 import { createPortal } from 'react-dom';
 import { RaRecord, TitleComponent, warning } from 'ra-core';
 
+import { PageTitle } from './PageTitle';
 import { PageTitleConfigurable } from './PageTitleConfigurable';
 
 export const Title = (props: TitleProps) => {
-    const { defaultTitle, title, preferenceKey, ...rest } = props;
+    const {
+        defaultTitle,
+        title,
+        preferenceKey,
+        nonConfigurable,
```

Comment:
**suggestion**: I realize... instead of adding a new prop, we could check if `preferenceKey` is false and render `<PageTitle>` or `<PageTitleConfigurable>` accordingly